templates/macros/filters/checkbox.html

```diff
@@ -33,3 +33,15 @@
   </fieldset>
 </div>
 {% endmacro %}
+
+{% macro checkbox_dropdown_multiple(name, label, selected={}, options={}) %}
```

Comment:
Other than by reading the source, it's not completely clear how `checkbox_dropdown` and `checkbox_dropdown_multiple` are different. We might be able to think of clearer names, or add comments.